database/src/main/postgres/flows/V1.7.1__flows.alter.ddl

```diff
@@ -14,4 +14,6 @@
  */
 
 ALTER TABLE flows.flows
-    ADD COLUMN IF NOT EXISTS fk_primary_partitioning BIGINT;
+    ALTER COLUMN fk_primary_partitioning SET NOT NULL;
+
+CREATE UNIQUE INDEX  IF NOT EXISTS unq_flows ON flows.flows (fk_primary_partitioning);
```

Comment:
Let's not modify existing migration SQL/DDL scripts please. Flyway won't be able to load it continuously - let's create a new file instead.

Reply:
That was actually a mistake, because I don't remember editing that file.

Reply:
Please rollback.